Frontend/src/pages/HomePage.tsx

```
@@ -333,47 +334,111 @@ function TrendingNichesSection() {
     </div>
   );
 }
-
+{/* Edited the cards for "why choose inpact ai? " */}
+{/* added under glow and 28 tiled hover effect-----by Praneet*/}
```

Biome (1.9.4)

[error] This block statement doesn't serve any purpose and can be safely removed.

Standalone block statements without any block-level declarations are redundant in JavaScript and can be removed to simplify the code.

(lint/complexity/noUselessLoneBlockStatements)